libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/xpub.syncing.integration.test.ts

```diff
@@ -55,8 +52,7 @@ describe("xpub integration sync", () => {
       addresses: 15,
       balance: 12688908,
       network: coininfo.bitcoin.main.toBitcoinJS(),
-      coin: "btc",
-      explorerVersion: "v3",
+      currencyId: "bitcoin",
```

Comment:
Any reason we're keeping this commented ? Can we delete it ?

Reply:
Yes, we want to keep the data but not run it during integration tests in the CI, it takes too much time